pkg/terminal/quickstart_p400.go

```diff
@@ -0,0 +1,132 @@
+package terminal
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/stripe/stripe-cli/pkg/config"
+	"github.com/stripe/stripe-cli/pkg/terminal/p400"
+)
+
+// QuickstartP400 runs the quickstart interactive prompt sequence to walk the user through setting up a P400 reader
+func QuickstartP400(cfg *config.Config) error {
+	tsCtx := SetTerminalSessionContext(cfg)
+
+	// reset the reader's state on SIGINT
+	interruptChannel := make(chan os.Signal, 1)
+	signal.Notify(interruptChannel, os.Interrupt)
+
+	go func() {
+		// block unless SIGINT occurs
+		<-interruptChannel
+		p400.ClearReaderDisplay(tsCtx)
+		os.Exit(1)
+	}()
+
+	tsCtx, err := p400.RegisterAndActivateReader(tsCtx)
+
+	if err != nil {
+		switch e := err.Error(); e {
+		case p400.ErrRegisterReaderFailed.Error():
+			return fmt.Errorf(err.Error())
+
+		case p400.ErrActivateReaderFailed.Error():
+			return fmt.Errorf(err.Error())
+
+		case p400.ErrConnectionTokenFailed.Error():
+			return fmt.Errorf(err.Error())
+
+		case p400.ErrNewRPCSessionFailed.Error():
+			return fmt.Errorf(err.Error())
```

Comment:
Not a huuuuge change but you could use `fallthrough`: https://github.com/golang/go/wiki/Switch#fall-through

Reply:
neat, will look at this thanks!